R/ml_generalized_linear_regression.R

```diff
@@ -27,6 +28,7 @@ ml_generalized_linear_regression <-
            features,
            intercept = TRUE,
            family = gaussian(link = "identity"),
+           weights_column = NULL,
```

Comment:
Given that other parameters here use `dot.case` rather than `snake_case`, should this be `weights.column`?